docs/_plugins/search_index.rb

```diff
@@ -9,10 +9,17 @@
 SEARCH_URL = (ENV["SEARCH_ORIGIN"] || 'https://search.modelshub.johnsnowlabs.com') + '/'
 ELASTICSEARCH_INDEX_NAME = ENV["ELASTICSEARCH_INDEX_NAME"] || 'models'
 
+WEBSITE = ENV["WEBSITE"] || ""
+
 OUTDATED_EDITIONS = ['Spark NLP 2.0', 'Spark NLP 2.1', 'Healthcare NLP 2.0']
+SPARK_NLP_ORIGIN = "https://sparknlp.org"
+JOHNSNOWLABS_ORIGIN = "https://nlp.johnsnowlabs.com"
 
 $remote_editions = Set.new
-
+if WEBSITE.empty?
```

Comment:
It will break a local server.

Reply:
You could pass `WEBSITE` in environment variable. Alternative it using default value, but which should we use default ?

Reply:
I mean people who edit the website will face this problem.
This variable is required in required for Elasticsearch. We could also check if ELASTICSEARCH_URL is not empty and only in this case stop the execution.

Reply:
Ok. I added the check only when ELASTICSEARCH_URL is available